spec/system/cohorts/show_spec.rb

```diff
@@ -27,6 +27,31 @@
   let!(:team_2) { create :team_with_students, cohort: cohort }
   let!(:team_3) { create :team_with_students, cohort: cohort_ended }
 
+  let!(:evaluation_criterion) { create :evaluation_criterion, course: course }
+
+  let!(:level_1) { create :level, :one, course: course }
+  let!(:level_2) { create :level, :two, course: course }
+
+  let!(:target_group_l1) { create :target_group, level: level_1 }
+  let!(:target_group_l2) { create :target_group, level: level_2 }
+
+  let!(:target_l1) do
+    create :target,
+           target_group: target_group_l1,
+           role: Target::ROLE_STUDENT,
+           evaluation_criteria: [evaluation_criterion],
+           milestone: true,
+           milestone_number: 1
+  end
+  let!(:target_l2) do
+    create :target,
+           target_group: target_group_l2,
+           role: Target::ROLE_STUDENT,
+           evaluation_criteria: [evaluation_criterion],
+           milestone: true,
+           milestone_number: 2
+  end
+
```

Comment:
You are not using any of it in the spec

Reply:
Because to render the student distribution, we need some milestone targets. The spec should be expanded.